src/routing_node.rs

```diff
@@ -623,18 +623,17 @@ impl RoutingNode {
         let threshold = self.group_threshold();
         debug!("Accumulator threshold is at {:?}", threshold);
 
-        let claimant : NameType = match *signed_message.claimant() {
+        let claimant: NameType = match *signed_message.claimant() {
             Address::Node(ref claimant) => claimant.clone(),
             Address::Client(_) => {
-                error!("Claimant is a Client, but passed into accumulator for a group. dropped.");
-                debug_assert!(false);
+                error!("Claimant is a Client, but passed into accumulator for a group, dropping.");
+                // debug_assert!(false);
```

Comment:
why uncomment the debug_assert!(false) ? is it then not better to remove it? or do you want it restored?